flatdata-generator/tests/generators/cpp_expectations/archives/namespaces.h

```diff
@@ -502,7 +503,7 @@ std::string STemplate< Member >::describe( ) const
 }
 } // namespace n
 
-namespace n { 
+namespace n {
```

Comment:
Pretty mixed style (also previously): Sometimes multiple in one line, sometimes '{' in the same line, sometimes in the next

Reply:
Should I address it somehow?

Reply:
No, just an observation.